eng/scripts/compatibility/Check-AOT-Compatibility.ps1

```diff
@@ -1,15 +1,49 @@
 param(
```

Comment:
Do you intend to keep this as a script as opposed to moving it into the azsdk cli?

If we do keep it as a script we should consider changing the args to take a package path instead of a service directory and package name.